Packs/CortexXDR/ReleaseNotes/6_1_95.md

```diff
@@ -1,9 +1,8 @@
-
 #### Integrations
 
 ##### Palo Alto Networks Cortex XDR - Investigation and Response
 
 - Updated the Docker image to: *demisto/python3:3.11.10.116949*.
 - Added two new optional arguments to the ***xdr-run-script-commands-execute*** command to fix issues caused by special characters:
   - ***is_raw_command*** - Can be set to `true` to prevent the splitting of the ***command*** argument value by commas. Useful when using `||`, `&&`, `;` separators for controlling the flow of multiple commands.
```

Comment:
```suggestion
  - *is_raw_command* - Can be set to `true` to prevent the splitting of the *command* argument value by commas. Useful when using `||`, `&&`, `;` separators for controlling the flow of multiple commands.
```